README.md

```diff
@@ -21,11 +26,12 @@ Further information on the Azure SQL binding for Azure Functions is also availab
       - [Docker container](#docker-container)
       - [Azure SQL Database](#azure-sql-database)
     - [SQL Setup](#sql-setup)
-    - [Create .NET Function App](#create-net-function-app)
+    - [Create a Function App](#create-a-function-app)
     - [Configure Function App](#configure-function-app)
   - [Tutorials](#tutorials)
-    - [Input Binding Tutorial](#input-binding-tutorial)
-    - [Output Binding Tutorial](#output-binding-tutorial)
+    - [.NET functions](#net-functions)
+        - [Input Binding Tutorial](#input-binding-tutorial)
+        - [Output Binding Tutorial](#output-binding-tutorial)
```

Comment:
Can we add links to the Javascript and Python sections as well?

Reply:
+ 1 for adding links to other language sections